src/NuGet.Core/NuGet.Common/Errors/NuGetLogCode.cs

```diff
@@ -159,16 +168,6 @@ public enum NuGetLogCode
         NU1605 = 1605,
 
         /// <summary>
-        /// Circular dependency.
-        /// </summary>
-        NU1606 = 1606,
-
-        /// <summary>
-        /// Version conflict.
-        /// </summary>
-        NU1607 = 1607,
-
-        /// <summary>
```

Comment:
Should we leave  comment here asking people to not create these?

Reply:
I'm with @mishra14 here, we probably should delete these but rather mark them as deprecated.

Reply:
I'll add comments